sql/core/src/test/scala/org/apache/spark/sql/execution/RowFormatConvertersSuite.scala

```diff
@@ -58,6 +58,41 @@ class RowFormatConvertersSuite extends SparkPlanTest with SharedSQLContext {
     assert(!preparedPlan.outputsUnsafeRows)
   }
 
+  test("coalesce can process unsafe rows") {
+    val plan = Coalesce(1, outputsUnsafe)
+    val preparedPlan = sqlContext.prepareForExecution.execute(plan)
+    assert(getConverters(preparedPlan).size === 1)
+    assert(preparedPlan.outputsUnsafeRows)
+  }
```

Comment:
oh sorry, I thought it was `org.apache.spark.sql.catalyst.expressions.Coalesce`, nvm